docs/source/tutorials/Install Myria3D on WSL2 with CUDA support.md

```diff
@@ -0,0 +1,132 @@
+# Install Myria3D on WSL2 with CUDA support
```

Comment:
Could you please reference this page in the index.st page, alonside the other tutorials of the Getting Started section?
Somewhere after [this line](https://github.com/jistiak/myria3d/blame/main/docs/source/index.rst#L13). To facilitate reference in the doc, you may rename this file itself something like `install_on_wsl2.md`.

Since there is already a `setup_install.md` file, could you please rename it to something like `install_on_linux.md` and its title to "Install Myria3D on Linux"